### DISCUSSION TITLE
Is there a way to make appear the _command palette_ beside `Ctrl+Shift+P` ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [Acode-Foundation/Acode#1018] [Q&A]
Hi,

is there a way (display a button or something else) to make appear the _command palette_ beside the `Ctrl+Shift+P` shortcut ?
( useful when you don't have a keyboard, (mobile phone ) )

Thanks

Out of topics, amazing software !

**Comment by bajrangCoder** [comment]
You can add legacy command palette icon in quick tools from Settings>App Settings>Shortcuts

**Reply by BobSquarePants** [reply]
Thanks @bajrangCoder , it worked,
For others that will face the same issue, I had to go to
>Settings>App Settings>Shortcuts **buttons** <--

Then inside that panel you can click on any icon and look for the one named _COMMAND PALETTE_ (positioned in this version third from the last.)

Cheers
